### PR TITLE
Menu pin

### PR DESCRIPTION
**Pull request by kewde**

### CODE REVIEW

**Comment by kewde** [comment]
@allienworks please take a look at the cold stake button in the widget.

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,7 +1,10 @@
-<md-sidenav-container fullscreen>
-  <md-sidenav #sidenav mode="side" opened="true">
-
+<md-sidenav-container fullscreen [ngClass]="{'menu-right': (!sideMenu && !window.isXS), 'menu-left': (window.isXS && showNav)}">
```
Comment (rynomster) [inline src/app/app.component.html]: these lines are waaaay too long :o

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+  <md-sidenav #sidenav mode="side" opened="true" [ngClass]="{'md-sidenav-left': (!isPinned && !window.isXS), 'hide-sidnav': (window.isXS && showNav)}"
```
Comment (rynomster) [inline src/app/app.component.html]: also very long line, can be shorter line

File: src/app/app.component.html [hunk 3]
```diff
@@ -70,14 +74,14 @@
               <a routerLinkActive="active" [routerLink]="walletInitialized ? '/settings' : null">
                 <md-list-item>
                   <md-icon fontSet="ncIcon" fontIcon="nc-grid-48"></md-icon>
-                  <span class="menu-item">Settings</span>
+                  <span class="menu-item" *ngIf="isPinned">Settings</span>
```
Comment (rynomster) [inline src/app/app.component.html]: We can revert this line

File: src/app/app.component.ts [hunk 4]
```diff
@@ -80,6 +93,22 @@ export class AppComponent implements OnInit {
...
+    if (value === 'toggle' && !this.window.isXS) {
+      this.sideMenu = !this.sideMenu;
+      return;
+    }
+    if (!this.sideMenu) {
+      this.isPinned = value === 'enter';
+    }
+  }
+
+  toggleSideNav(sidNav: MdSidenav) {
```
Comment (rynomster) [inline src/app/app.component.ts]: what is a sidNav?

File: src/app/layouts/side-nav/side-nav.component.scss [hunk 5]
```diff
@@ -0,0 +1,118 @@
+@import "./src/assets/_config"; // import shared colors etc.
```
Comment (rynomster) [inline src/app/layouts/side-nav/side-nav.component.scss]: is this file used at all?

Comment (allienworks) [reply src/app/layouts/side-nav/side-nav.component.scss]: Yes, it is. It cointains all the variables (color, fonts).

Comment (rynomster) [reply src/app/layouts/side-nav/side-nav.component.scss]: haha I don't mean config, I mean `src/app/layouts/side-nav/side-nav.component.scss`

Comment (allienworks) [reply src/app/layouts/side-nav/side-nav.component.scss]: AFAIK this component wasn't used at all, but @kewde IMO moved the sidebar nav there to make use of it (+ originally the sidebar was in the `app.component` there it didn't make any sense).

File: src/app/app.component.ts (hunk 4 above)
Comment (vikas-cis) [reply src/app/app.component.ts]: It is the object that we referenced for toggling

File: src/app/layouts/side-nav/side-nav.component.scss (hunk 5 above)
Comment (kewde) [reply src/app/layouts/side-nav/side-nav.component.scss]: Yeah the sidenav css is scattered around, I moved some of it to the component..

**Comment by rynomster** [comment]
so what now?

**Comment by allienworks** [comment]
Shouldn't we close this ancient PR and start again from scratch on `dev` branch?

**Comment by allienworks** [comment]
Ancient, outdated.

**Pull request closed without merging**